docs/configuration.rst

```diff
@@ -1,12 +1,245 @@
+
+Basic authentication and SSL
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+By default all traffic to Elasticsearch is via unencrypted HTTP because
+Elasticsearch does not come with built-in support for SSL unless you pay for
+the expensive enterprise X-Pack addition. A cheap alternative to X-Pack, is to
+simply setup a proxy (e.g. nginx) on each node with SSL and
+HTTP basic authentication support.
+
+Following is an example of how you configure Invenio to use SSL and Basic
+authentication when connecting to Elasticsearch:
+
+.. code-block:: python
+
+    params = dict(
+        port=443,
+        http_auth=('myuser', 'mypassword'),
+        use_ssl=True,
+    )
+    SEARCH_ELASTIC_HOSTS = [
+        dict(host='node1', **params),
+        dict(host='node2', **params),
+        dict(host='node3', **params),
+    ]
+
+Self-signed certificates
+~~~~~~~~~~~~~~~~~~~~~~~~
+In case you are using self-signed SSL certificates on proxies in front of
+Elasticsearch, you will need to provide the ``ca_certs`` option:
+
+.. code-block:: python
+
+    params = dict(
+        port=443,
+        http_auth=('myuser', 'mypassword'),
+        use_ssl=True,
+        ca_certs='/etc/pki/tls/mycert.pem',
+    )
+    SEARCH_ELASTIC_HOSTS = [
+        dict(host='node1', **params),
+        # ...
+    ]
+
+**Disabling SSL certificate verification**
+
+.. warning::
+
+    We **strongly discourage** you to use this method. Instead, use the method
+    with the ``ca_certs`` option documented above.
+
+    Disabling verification of SSL certificates will e.g.  allow
+    man-in-the-middle attacks and give you a false sense of security (thus you
+    could simply use plain unencrypted HTTP instead).
+
+If you are using a self-signed certificate, you may also disable verification
+of the SSL certificate, using the ``verify_certs`` option:
+
+.. code-block:: python
+
+
+    import urllib3
+    urllib3.disable_warnings(
+        urllib3.exceptions.InsecureRequestWarning
+    )
+
+    params = dict(
+        port=443,
+        http_auth=('myuser', 'mypassword'),
+        use_ssl=True,
+        verify_certs=False,
+        ssl_show_warn=False, # only from 7.x+
+    )
+    SEARCH_ELASTIC_HOSTS = [
+        dict(host='node1', **params),
+        # ...
+    ]
+
+Above example will also disable the two warnings (``InsecureRequestWarning``
+and a ``UserWarning``) using the ``ssl_show_warn`` option and urllib3 feature.
+Again, we **strongly discourage** you from using this method. The warnings are
+there for a reason!
+
+Other host options
+~~~~~~~~~~~~~~~~~~
+For a full list of options for configuring the hosts, see the connection
+classes documentation:
+
+- :py:class:`elasticsearch.connection.Urllib3HttpConnection` (default)
+- :py:class:`elasticsearch.connection.RequestsHttpConnection`
+
+Other options include e.g.:
+
+- ``url_prefix``
+- ``client_cert``
+- ``client_key``
+
+
+Client options
+--------------
+More advanced options for the Elasticsearch client is configured via the
```

Comment:
is configured -> are configured. Now I'm suspicious if it is not on purpose

Reply:
You're right - it's me who is bad at getting is/are right when I'm writing ;-)